tests/core/assertions.test.js

```diff
@@ -13,17 +17,17 @@ afterAll(() => {
 
 beforeEach(() => {})
 
-test('should allow to count object properties', () => {
+test('should allow to build an array of object properties', () => {
```

Comment:
I'd suggest to add a test where we pass an invalid object, and also a test where properties contain dots or spaces.

Reply:
done

Reply:
I've added some, may need rework depending on what we decide :)